Source/diablo.cpp

```diff
@@ -890,6 +890,8 @@ void PressKey(int vkey)
 			diablo_hotkey_msg(3);
 		}
 		if (vkey == VK_RETURN) {
+			if (GetAsyncKeyState(VK_MENU) & 0x8000)
+			dx_reinit();
 			control_type_message();

```

Comment:
this dhoudn't be called if alt is pressed at the same time

Reply:
Do you mean dx_reinit()?

Reply:
control_type_message, this will call both, only one function should be called by the key press

Reply:
I changed it to following:
if (vkey == VK_RETURN) {
			control_type_message();
			if (GetAsyncKeyState(VK_MENU) & 0x8000)
				dx_reinit();

Reply:
Looks that code better?

Reply:
no XD

Reply:
```c++
if (GetAsyncKeyState(VK_MENU) & 0x8000)
    dx_reinit();
else
    control_type_message();
```

Reply:
Oh I already separated in another way. Is it also okay?

Reply:
no ...

Reply:
So happy now?

Reply:
no ... look at the changes, you butchered it XD

Reply:
Ahh. you mean the else indentation.